examples/pytorch/speech-pretraining/run_wav2vec2_pretraining_no_trainer.py

```diff
@@ -247,6 +247,22 @@ def parse_args():
         "--hub_model_id", type=str, help="The name of the repository to keep in sync with the local `output_dir`."
     )
     parser.add_argument("--hub_token", type=str, help="The token to use to push to the Model Hub.")
+    parser.add_argument(
+        "--mask_time_prob",
+        type=float,
+        help=(
```

Comment:
Could we add the default value for this arg for consistency with the others?
```suggestion
        type=float,
        default=None,
        help=(
```